storage/blockdevice/COMPONENT_QSPIF/include/QSPIF/QSPIFBlockDevice.h

```diff
@@ -376,6 +376,9 @@ class QSPIFBlockDevice : public mbed::BlockDevice {
 
     bool _needs_fast_mode;
 
+    // S25FS512S needs a quirk
```

Comment:
In commit "QSPIFBlockDevice: Add quirk for inconsistent CR3NV register value", we say "But it is a known issue". Can you link to errata or mailing list posts for this in the commit message?